src/ds5/ds5-auto-calibration.h

```diff
@@ -39,8 +55,30 @@ namespace librealsense
         void undistort(uint8_t* img, const rs2_intrinsics& intrin, int roi_ws, int roi_hs, int roi_we, int roi_he);
         void find_z_at_corners(float left_x[4], float left_y[4], rs2_frame_queue* frames, float left_z[4]);
         void get_target_dots_info(rs2_frame_queue* frames, float dots_x[4], float dots_y[4], rs2::stream_profile & profile, rs2_intrinsics & fy, int progress, update_progress_callback_ptr progress_callback);
+        void change_preset_and_stay();
+        void restore_preset();
+        uint16_t calc_fill_rate(const rs2_frame* f);
+        void fill_missing_data(uint16_t data[256], int size);
+        void collect_depth_frame_sum(const rs2_frame* f);
 
         std::vector<uint8_t> _curr_calibration;
         std::shared_ptr<hw_monitor>& _hw_monitor;
+
+        bool _preset_change = false;
+        preset _old_preset_values;
+        rs2_rs400_visual_preset _old_preset;
+
+        std::string _json;
+        float _ground_truth_mm;
+        int _total_frames;
+        int _average_step_count;
+        int _collected_counter;
+        int _collected_frame_num;
+        double _collected_sum;
+        auto_calib_action _action;
+        interactive_calibration_state _occ_state;
```

Comment:
Rename `occ` to `interactive` or similar

Reply:
Done.